packages/compass-aggregations/src/utils/insights.ts

```diff
@@ -25,19 +25,27 @@ const SIGNALS: Record<string, Signal> = {
     primaryActionButtonLink: ATLAS_SEARCH_LP_LINK,
     primaryActionButtonLabel: 'Try Atlas Search',
   },
+  'lookup-in-stage': {
+    id: 'lookup-in-stage',
+    title: '$lookup usage',
+    description:
+      '$lookup operations can be resource intensive because they perform operations on two collections instead of one. In certain situations, consider embedding documents or arrays to enhance read performance.',
```

Comment:
[very grammary-nitty, feel free to ignore] I think technically, since it’s a compound adjective:

```suggestion
      '$lookup operations can be resource-intensive because they perform operations on two collections instead of one. In certain situations, consider embedding documents or arrays to enhance read performance.',
```

“Consider” and “In certain situations” are also redundant, so I’d maybe drop one of them:

```suggestion
      '$lookup operations can be resource-intensive because they perform operations on two collections instead of one. Consider embedding documents or arrays to enhance read performance.',
```

```suggestion
      '$lookup operations can be resource-intensive because they perform operations on two collections instead of one. In certain situations, embedding documents or arrays can enhance read performance.',
```

Reply:
> “Consider” and “In certain situations” are also redundant, so I’d maybe drop one of them

@Anemy as this was your suggested change and this was a blocker, can you give some feedback on this?

Reply:
For reference, before / after last update to this description:

```diff
- $lookup operations can be resource intensive because they perform operations on two collections instead of one. Consider embedding documents or arrays to increase read performance.
+ $lookup operations can be resource intensive because they perform operations on two collections instead of one. In certain situations, consider embedding documents or arrays to enhance read performance.
```

Reply:
I should mention I still feel this insight is going to be more of a distraction than a help to most folks.